app/services/delivery_partners/participants_filter.rb

```diff
@@ -61,7 +63,13 @@ def filter_role(scoped, role)
       end
     end
 
+    def filter_out_cohorts(scoped)
+      scoped.joins(:cohort).where("start_year <= ?", LATEST_COHORT_TO_RETURN)
```

Comment:
Should we do this in `app/services/delivery_partners/induction_records_query.rb` instead?

We're seeing timeouts on the AB participants filter which has a similar logic to DP filter.. please see https://dfedigital.atlassian.net/browse/CPDLP-4045. 

Maybe DP participants has lesser records in comparison with AB, so no timeout are raised so we should be fine doing it here. Needs some assessment.

Reply:
I think it makes sense to keep the filtering all in one place; if I was looking for it I would look in `ParticipantsFilter`. As we're not introducing new functionality here (we already support filtering on cohort) it shouldn't cause any new performance issues - happy to discuss if you feel differently though?